wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -272,4 +276,10 @@ public void setScreenOrientation(ScreenOrientation orientation) throws Exception
     public void unsetScreenOrientation() throws Exception {
         getUiDevice().unfreezeRotation();
     }
+
+   public void uiDevicePerformLongClick(UiObject view, int steps) throws Exception {
```

Comment:
Is it having an other method where the step is predefined within this base class for convenience?  Maybe three seconds?